xdsl/pattern_rewriter.py

```diff
@@ -308,6 +308,18 @@ def move_region_contents_to_new_regions(self, region: Region) -> Region:
             )
         return Rewriter.move_region_contents_to_new_regions(region)
 
+    def iter_affected_ops(self) -> Iterable[Operation]:
```

Comment:
Can you add documentation to these two functions?

Reply:
- [ ] add documentation